### PR TITLE
Org readme

### PR DESCRIPTION
**Pull request by abbeyperini**
[#1](https://github.com/Virtual-Coffee/.github/issues/1)

Adding:
 A welcome message
 Probably the full logo
 Some short FAQs (what is this, how do I join, etc)
 Links to resources like Code of Conduct, Contributing Guide, links to the website
 A section about sponsorship

Bonus:
 some flair and a link to the youtube page

### CODE REVIEW

**Review by danieltott — CHANGES_REQUESTED** [review]
Thank you @abbeyperini !

This is a really good start! Only thing is - the org readMe goes into a folder called 'profile' - the root repo readme should be about the '.github' repo (it was pretty empty before).

**Comment by abbeyperini** [comment]
#2 updated to reflect requested changes

Thanks for catching that!

**Comment by BogDAAAMN** [comment]
> Bonus:
> some flair and a link to the youtube page

I like this!! I was thinking if there is a _Here are some of the things that sponsorship money will go to support_ list, maybe to support that we can also make a list (or add to this one) of the playlists that we have on yt. Like _here are the Lunch & Learn sessions_, _here are the lightning talks_ and so on.. 

What do you think? I think it's a nice way to showcase all the content we put out. Or it gets too crowded?

File: profile/README.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+Our Mission
+------
+Virtual Coffee's mission is to be a welcoming tech community that allows room for growth and mentorship at all levels, and to create meaningful opportunities for learning, leadership, and contribution for everyone.
+
+[Sponsor Us!](https://github.com/sponsors/Virtual-Coffee)
+------
+Here are some of the things that sponsorship money will go to support:
+
+* Twice-Weekly Coffee Chats
+* Member-led Presentations and Workshops
```
Comment (danieltott) [inline profile/README.md:17]: Let's add a link to the Lunch & Learn playlist: https://www.youtube.com/playlist?list=PLh9uT23TA65idCyc_orC85RefgY_-fKsG

File: profile/README.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+* A weekly [podcast](https://virtualcoffee.io/podcast/) sharing the stories of our members
+* A monthly [newsletter](https://virtualcoffee.io/newsletter/) with Virtual Coffee news and interesting things our members are doing
+* Our growing Slack community
+* The Virtual Coffee Co-Working Room
+* Lightning Talks
```
Comment (danieltott) [inline profile/README.md:22]: Maybe a link here to our previous lightning talks? https://www.youtube.com/playlist?list=PLh9uT23TA65infGqUUaVibI0IssI0G6NY

File: profile/README.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+* The Virtual Coffee Hacktoberfest Initiative
```
Comment (danieltott) [inline profile/README.md:23]: https://hacktoberfest.virtualcoffee.io

File: profile/README.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+[<img src="../TheJoyOfDevLifePreview.png" alt-text="A youtube video thumbnail showing 'Bekah Hawrot Wigel @BekahHW The Joy of Dev Life: A Bob Ross Guide' on a background of mountains and trees" width="400px">](https://www.youtube.com/c/virtualcoffeeio)
+
+FAQs :pencil:
+------
+### How do I become a member?
+To become a member of Virtual Coffee, all you need to do is attend [a Tuesday or Thursday Coffee](https://virtualcoffee.io/events/) and submit the form you'll receive at coffee.
+### What's the code of conduct?
+In order to create a welcoming and inclusive community, we pledge to make participation in our community a harassment-free experience for everyone, regardless of age, body size, visible or invisible disability, ethnicity, gender identity and expression, level of experience, education, socio-economic status, nationality, personal appearance, race, religion, or sexual identity and orientation.
+
+[Read more](https://github.com/Virtual-Coffee/virtualcoffee.io/blob/main/CODE_OF_CONDUCT.md) about how we enforce this and cultivate a community that assumes good intent.
```
Comment (danieltott) [inline profile/README.md:38]: Let's change this link to https://virtualcoffee.io/code-of-conduct

**Review by danieltott — CHANGES_REQUESTED** [review]
Alllllright - now that I'm in front of my machine, just a couple link changes I think (incorporating [some of Bogdan's suggestions too](https://github.com/Virtual-Coffee/.github/pull/2#issuecomment-921945597))

File: profile/README.md (hunk 4 above)
Comment (abbeyperini) [reply profile/README.md:38]: Ah, man. I had that first and second guessed myself.

**Comment by abbeyperini** [comment]
Added all the things above and a link to the blog about how the co-working room works!

**Review by danieltott — APPROVED** [review]
This is amazing, thank you so much @abbeyperini

**Pull request merged**